src/main/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/messages_ru.properties

```diff
@@ -1,5 +1,6 @@
 at.clause.order=Теги должны располагаться по порядку: ''{0}''.
 invalid.position=Комментарии Javadoc размещены не в том месте.
+javadoc.asterisk.indentation=Leading asterisk has incorrect indentation, should be {0}.
```

Comment:
I need one help here, then all the implementation work of this check will be done. 

The translation of this error message is pending. The messages are not simply translated, they're encoded in different kind of format. I tried researching about it but I was not able to find a proper answer to this question. Can you tell me how we can translate the error messages to different languages with encoded format?

The translation for other languages was done by previous contributor of this issue but it didn't had translation for this specific language. @romani @nrmancuso

Reply:
We just use Google translate for languages we don’t speak, this is good enough until a native speaker noticed and improves them.

Reply:
So I guess I'll leave it as it is 👍